packages/cli/src/commands/styles/utils/publish.main.ts

```diff
@@ -21,39 +22,71 @@ export const generateToken = (name: string) => {
 export const filterFunctions: NodeFilter[] = [];
 export const designTokenFunctions: DesignTokenFilter[] = [];
 
-const colorDesignTokenizer: DesignTokenFilter = (node: NodeDocument): DesignToken => {
-  colorToHex(node.fills[0].color);
-  const colorToken: DesignToken = {
-    type: 'color' ,
-    name: node.name,
-    token: generateToken(node.name),
-    value: colorToHex(node.fills[0].color)
-  };
-  return colorToken;
-};
 
-const colorFilter: NodeFilter = (data: NodeDocument, designTokenFilterFn: DesignTokenFilter): DesignToken => {
-  if(data.type == 'RECTANGLE') {
-    data.type = 'color';
+export const processStyleNodes = (data: NodeDocument, type: string): DesignToken[] | DesignToken | undefined => {
+  if(!data) return undefined;
+  switch(type){
+    case 'FILL':
+      // simple
+      return colorDesignTokenizer(data);
+    case 'TEXT':
+      return typographyTokenizer(data);
+    case 'EFFECT':
+      return processElevationTokenizer(data);
+    default:
+      return undefined;
```

Comment:
we can remove `return undefined` and add break to the default case. The break will return undefined

```
default:
    break
```